CONTRIBUTING.md

```diff
@@ -9,13 +9,28 @@ contributor license agreement.
 
 ## Submitting Changes
 
-Submit a pull request rebased on top of master
+Submit a pull request rebased on top of master.
 
- * Include a descriptive commit message.
- * Changes contributed via pull request should focus on a single issue at a time.
+Workflow:
 
-At this point you're waiting on us. We like to at least comment on pull requests within one week
+1. **Fork repo on GitHub:** Visit the [`stellar/stellar-core`](https://github.com/stellar/stellar-core) repository and click on the `Fork` button to create a copy of the repository under your GitHub account.
+1. **Clone Locally:**
+`git clone https://github.com/stellar/stellar-core.git`
+1. **Set Your Fork as remote upstream:** `cd stellar-core &&
```

Comment:
I think this is reversed. Usually, the original forked repo is refereed to as upstream. I think we should change this so that you clone your fork as origin and set the original repo as upstream. We can change the rebase command to `git fetch upstream && git rebase upstream/master`